### DISCUSSION TITLE
Record resolving in vcs system

### DISCUSSION THREAD

**Discussion by corporal9736** [prisma/prisma#23610] [Q&A] [answered]
### Question

I have multiple databases initialized at the same baseline state. If I want to correctly apply migrations after, as I understand, I need to run `npx prisma migrate resolve --applied 0_init` for every database.
Based on the assumption that all database should be in the same state, can we just add something like "applied migrations" in migration_lock.toml, so that resovling can be recorded in vcs like git and easily applied among different databases?
I don't know if this is possible, thanks in advance.

**Comment by ludralph** [comment, marked as answer]
Hi @corporal9736 👋 

Prisma Migrate system does not currently support recording applied migrations in the `migration_lock.toml` or any other file that could be version controlled. The state of the migrations is managed within the database itself, in the `_prisma_migrations` table.
If you want to apply the same migrations to multiple databases, you would need to run the `npx prisma migrate resolve --applied 0_init` command for each database individually.
However, if you want to see this particular feature, you can create a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) documenting your use case so our engineers can take a look.


If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
